templates/seL4VMParameters.template.c

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2023, DornerWorks
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <camkes.h>
+#include <arm_vm/gen_config.h>
+
+/*- set vm_address_config = configuration[me.name].get('vm_address_config') -*/
+/*- set vm_image_config = configuration[me.name].get('vm_image_config') -*/
+/*- set linux_address_config = configuration[me.name].get('linux_address_config') -*/
+/*- set linux_image_config = configuration[me.name].get('linux_image_config') -*/
+
+/*- if vm_address_config -*/
+
+unsigned long ram_base = /*? vm_address_config.get('ram_base') ?*/;
```

Comment:
Actually, these are all constants and they could be put into a ro-segement.
```suggestion
const unsigned long ram_base = /*? vm_address_config.get('ram_base') ?*/;
```

Reply:
Done.